src/main/java/edu/depaul/cdm/se452/d2l_mock/user_permission/User.java

```diff
@@ -0,0 +1,25 @@
+package edu.depaul.cdm.se452.d2l_mock.user_permission;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class User {
```

Comment:
The Professor did an example in the last lecture that relates to this class. There was Employee, SalaryEmployee, HourlyEmployee and it made 3 different tables for each one. Then he was able to treat them as one in the code. I think we can do something similar with User, Professor, and Student.

Reply:
Okay, I'll go back to that example and check it out!